src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/ArtifactoryGradleBuild/help-tasks.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify a list of Gradle tasks to be invoked.
```

Comment:
Do they have to be comma separated, on new lines, or any other type of delimiter?  Is this an array?

Reply:
They are space-separated. I will include that. Thanks